packages/synapse-react-client/jest.config.cjs

```diff
@@ -4,6 +4,8 @@ const esModules = [
   'lodash-es',
   'nanoid',
   'mui-one-time-password-input',
+  'p-limit',
+  'yocto-queue',
```

Comment:
p-limit is a new dependency. yocto-queue is an esm-only dependency pulled in by yocto-queue